smpe/ZOSMF/smpe_workflow.xml

```diff
@@ -280,6 +280,17 @@
 		<string multiLine="false" valueMustBeChoice="false">
 			<validationType>DSMEMBERNAME</validationType>
 		</string>
+	</variable>
+		<variable name="sysmodsCheck" scope="instance" visibility="public">
+		<label>CHECK operand</label>
+		<abstract>A good way to see which SYSMODs are included before you actually install them is with the CHECK operand.</abstract>
```

Comment:
This comment makes sense when installing large service packs, but not so much for Zowe. For the FMID it is just the FMID, and since Zowe PTFs replace everything, only the last one matters. Please rewrite to say the CHECK operand lets SMP/E run through the motions without doing actual copies, thus checking that setup looks good.